.github/workflows/release.yaml

```diff
@@ -20,11 +20,8 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: Select charmhub channel
-        uses: canonical/charming-actions/upload-charm@2.1.1
-        id: channel
```

Comment:
Actually other repos uses "channel" here:

      - name: Select charmhub channel
        uses: canonical/charming-actions/channel@2.1.1
        id: channel

Is it necessary here too?

Reply:
No, because we're hard coding the channel in the next step anyway - so it negates the need for the `channel`action :)

Reply:
TIL. TL;DR: not necessary here as we have hardcoded channel name in line "31 28":
>  channel: "dpe/edge"

Necessary here only in the case:
> channel: "${{ steps.channel.outputs.name }}"
See https://github.com/canonical/charming-actions/tree/main/channel for more details.